test/Test/Utils.purs

```diff
@@ -0,0 +1,28 @@
+module Test.Utils where
+
+import Prelude
+
+import Concur.Core (Widget)
+import Concur.Core.Types (WidgetStep(..), unWidget)
+import Control.Monad.Free (runFreeM)
+import Control.Monad.Writer.Trans (runWriterT, tell)
+import Data.Array (singleton)
+import Data.Tuple (Tuple(..))
+import Effect.Aff (Aff)
+import Effect.Aff.Class (liftAff)
+import Effect.Class (liftEffect)
+
+
+-- Evalutates Widget to Aff
```

Comment:
This looks very useful, it can probably go in `Concur.Core.Discharge` which has various strategies to extract views from Widgets.